Hackney.Shared.HousingSearch.Tests/Models/TenureTests.cs

```diff
@@ -0,0 +1,33 @@
+using FluentAssertions;
+using Xunit;
+using AutoFixture;
+using Hackney.Shared.HousingSearch.Gateways.Models.Tenures;
+
+namespace Hackney.Shared.HousingSearch.Tests
+{
+    public class TenureTests
+    {
+        private Fixture _fixture = new Fixture();
+
+        [Fact]
+        public void DomainTenureHasPropertiesSet()
+        {
+            //arrange    
+            var aQueryableTenure = _fixture.Create<QueryableTenure>();
+
+            //act
+            var createdDomainTenure = aQueryableTenure.Create();
+
+            //assert
+            createdDomainTenure.Id.Should().Be(aQueryableTenure.Id);
+            createdDomainTenure.TempAccommodationInfo.BookingStatus.Should().Be(aQueryableTenure.TempAccommodationInfo.BookingStatus);
```

Comment:
This is probably unnecessary since `TempAccommodationInfo` is covered below.

Reply:
I wanted to be double sure XD
JK, addressed in [489cb25](https://github.com/LBHackney-IT/housing-search-shared/pull/73/commits/489cb25724400385398549e75d24db37fe081d37).